src/diet.ts

```diff
@@ -501,6 +503,23 @@ export function runDiet(): void {
   const { bestSpleenItem } = getBestSpleenItems();
   const embezzlers = embezzlerCount();
   if (embezzlers) {
+    if (
+      mySpleenUse() + 2 <= spleenLimit() &&
+      !get("beGregariousCharges") &&
+      !(
+        get("beGregariousFightsLeft") &&
+        get("beGregariousMonster") === $monster`Knob Goblin Embezzler`
+      )
+    ) {
+      const value = expectedGregs() * 25000;
```

Comment:
Is this meant to be the expected meat from an Embez fight? Is this accounting for the diff between an Embez fight and a barf fight? I'm not sure I'm making 25k+7k from my Embezzlers. It is totally good back of the envelope math though. I'm just surprised we don't have an EmbezzlerMPA var if we have a regular mpa var

Reply:
7k is pretty high an estimate for a barf-turn, but I guess it's start of the day. EmbezzlerMPA is very hard to estimate and varies a lot more over the course of a day than barf MPA. I consider this a decent temproary measure.